cinema/models.py

```diff
@@ -5,6 +5,32 @@ class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
+    actors = models.ManyToManyField("Actor")
```

Comment:
good practice when add an argument "related_name" in Many To Many Relation

Reply:
I'll fix